<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysiologyExplorerMainWindow</class>
 <widget class="QMainWindow" name="PhysiologyExplorerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QDockWidget" name="inputDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="scenarioGroupBox">
       <property name="title">
        <string>Choose Scenario</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="scenarioComboBox">
          <item>
           <property name="text">
            <string>Anaphylaxis</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="anaphylaxisGroupBox">
       <property name="title">
        <string>Anaphylaxis</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="obstructionGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Airway Obstruction</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Severity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="pqDoubleRangeWidget" name="severitySlider" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum" stdset="0">
              <number>1</number>
             </property>
             <property name="minimum" stdset="0">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addOstructionButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="epiButton">
             <property name="text">
              <string>Epi</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="engineGroupBox">
          <property name="title">
           <string>Engine Interaction</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Time step:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="timeStepBox">
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string> s</string>
               </property>
               <property name="minimum">
                <double>0.250000000000000</double>
               </property>
               <property name="maximum">
                <double>60.000000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="advanceButton">
             <property name="text">
              <string>Advance</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Speed:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="playButton">
               <property name="text">
                <string>Play</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="playbackSpeedBox">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="resetButton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputDockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QTextEdit" name="logOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqDoubleRangeWidget</class>
   <extends>QWidget</extends>
   <header>pqDoubleRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
